src/coingecko_api/base_client.py

```
@@ -0,0 +1,97 @@
+import asyncio
+import logging
+from typing import Dict, Any, Optional
+from dataclasses import dataclass, field
```

Ruff (0.11.9)

`dataclasses.field` imported but unused

Remove unused import: `dataclasses.field`

(F401)